playbook/app/pb_kits/playbook/pb_button/_button.jsx

```diff
@@ -15,6 +15,7 @@ type ButtonPropTypes = {
   data?: object,
   disabled?: boolean,
   fixedWidth?: boolean,
+  form?: string,
```

Comment:
@nehaabraham I made this optional since there will be many instances in Nitro which will not be sending this prop. Once this kit is converted to Typescript, not having the optional `?` will throw errors.